sormas-backend/src/main/java/de/symeda/sormas/backend/externaljournal/ExternalJournalService.java

```diff
@@ -315,7 +315,8 @@ public PatientDiaryRegisterResult registerPatientDiaryPerson(PersonDto person) {
 
 	/**
 	 * Check whether a person has valid data in order to be registered in the patient diary
-	 * 
+	 * NOTE: since CLIMEDO is only used in Germany, only German numbers are considered valid at the moment
+	 *
```

Comment:
We could for example state that a PatientDiary is climedo, and a SymptomJournal is a PIA related system. They also have sormas-sb now, but that is, as far as I know, just a variant of PIA...

Reply:
We always write code for SymptomJournals and PatienDiaries, but these two actually just refer to these two systems. For people to understand you comment, it should be clear that climedo is the only PatientDiary so far.

Reply:
I added clarifications in a javadoc on the `ExternalJournalService` class